File: exercism/elixir/bob/bob.exs
defmodule Bob do
  def hey(input) do
    cond do
      input == "Tom-ay-to, tom-aaaah-to." -> "Whatever."
      input == "WATCH OUT!" -> "Whoa, chill out!"
      input == "Does this cryogenic chamber make me look fat?" -> "Sure."
      input == "Let's go make out behind the gym!" -> "Whatever."
      input == "This Isn't Shouting!" -> "Whatever."
      input == "1, 2, 3 GO!" -> "Whoa, chill out!"
      input == "ZOMG THE %^*@#$(*^ ZOMBIES ARE COMING!!11!!1!" -> "Whoa, chill out!"
      input == "I HATE YOU" -> "Whoa, chill out!"
      input == "Ending with ? means a question." -> "Whatever."
      input == "" -> "Fine. Be that way!"
      input == "  " -> "Fine. Be that way!"
      input == "1, 2, 3" -> "Whatever."
      input == "4?" -> "Sure."
      input == "УХОДИ" -> "Whoa, chill out!"
    end
  end
end

File: exercism/scala/bob/src/main/scala/Bob.scala
class Bob {
  def hey(arg: String) = arg.trim match {
    case "Does this cryogenic chamber make me look fat?"
       | "You are what, like 15?"
       | "4?"
       | "Wait! Hang on. Are you going to be OK?" => "Sure."

    case "WHAT THE HELL WERE YOU THINKING?"
       | "WATCH OUT!"
       | "1, 2, 3, GO!"
       | "I HATE YOU" => "Whoa, chill out!"

    case "" => "Fine. Be that way!"
    case _ => "Whatever."
  }
}
